model/src/main/kotlin/LicenseFinding.kt

```diff
@@ -79,7 +61,7 @@ data class LicenseFinding(
  * Apply [detectedLicenseMapping] from the [org.ossreviewtoolkit.model.config.ScannerConfiguration] to any license
  * String.
  */
-private fun String.applyDetectedLicenseMapping(detectedLicenseMapping: Map<String, String>): String =
+fun String.mapLicense(detectedLicenseMapping: Map<String, String>): String =
```

Comment:
The new function name makes it less obvious that the function is only "detected" licenses. How about renaming the function to `mapDetectedLicense` instead? (Maybe the parameter name can then be shortened to just "mapping", which also make the tests a bit less verbose)

Reply:
I was actually tempted to do the "opposite": Keep the function name and also generalize the parameter name. There's actually nothing "detected license"-specific in that function, and maybe not even something "(any) license"-specific. It's just about folding string replacements. What *makes* the function "detected license"-specific is the *call* to it with a detected license mapping. So at the call sites, it's IMO obvious what this is about. The only reason I kept "License" as part of "mapLicense" as "map" alone is already taken, and you could argue that the way the regex is design makes it specific to the license replacement use-case.